tutorials/image/cifar10/cifar10_multi_gpu_train.py

```diff
@@ -169,7 +176,7 @@ def train():
             # Calculate the loss for one tower of the CIFAR model. This function
             # constructs the entire CIFAR model but shares the variables across
             # all towers.
-            loss = tower_loss(scope)
+            loss = tower_loss(scope, images, labels)
```

Comment:
Fixed, I am using slim.  My test (which my not be great) is I ran for 1000 steps with batch_size-128 on one GPU.  Then I ran batch_size=32 on multi-GPU with my original method and then the new method to verify the difference with the "wrong" original method I get a much lower loss than I get with the new method.  Maybe not perfect and this is also how it is done with slim.  Given queue readers on on the way out Using slim seems fine and I would use this over a manual split.  I could not find a better example.